### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by PayamZandiyeh**
I have made the necessary subfolders for different contributors to work on. Will it be possible to merge it to master if that's OK.

### CODE REVIEW

**Comment by Besler** [comment]
FYI future commits should stick to the software guidelines in the README.md file.

**Software Guidelines**
We are going to follow the ITK Style for software development. Those rules roughly manifest themselves as such:

Do not push to master

- All work is done on a seperate branch: git checkout -b my-topic
- Merge on GitHub after at least one review
    - Follow standard prefixes for commits:
    - BUG: Fix for runtime crash or incorrect result
    - COMP: Compiler error or warning fix
    - DOC: Documentation change
    - ENH: New functionality
    - PERF: Performance improvement
    - STYLE: No logic impact (indentation, comments)
    - WIP: Work In Progress not ready for merge

**Comment by Besler** [comment]
Also FYI you can request a review in the top right. Put my name in and I'll get a message!

File: Registration/3D2DRegistration/PythonImplementation/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# StereoFlouroscopyRegistration
```
Comment (Besler) [inline Registration/3D2DRegistration/PythonImplementation/README.md]: README should be specific to this directory. You shouldn't copy the old README

File: Registration/MRI2PseudoCT/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# StereoFlouroscopyRegistration
```
Comment (Besler) [inline Registration/MRI2PseudoCT/README.md]: See previous README comments

File: Registration/Segmentation/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# StereoFlouroscopyRegistration
```
Comment (Besler) [inline Registration/Segmentation/README.md]: See previous README comments

File: Registration/3D2DRegistration/PythonImplementation/SourceCode/ScrapProject.py [hunk 4]
```diff
@@ -0,0 +1 @@
+#%%
```
Comment (Besler) [inline Registration/3D2DRegistration/PythonImplementation/SourceCode/ScrapProject.py]: Could this be deleted?

**Review by Besler — CHANGES_REQUESTED** [review]
It's looking good! I can't make the comments on binary files, but could you do the following?

Remove all mhd/mha/pdf files? And remove them from history as well. See https://wikileaks.org/ciav7p1/cms/page_41123851.html for how to remove files from history. Note: you're rewriting history. Becareful with that power.

Typically, you don't version these files. You store them somewhere else and download them when you need them. And I don't think they're our files.

**Comment by PayamZandiyeh** [comment]
Hello Bryce, 
I have removed the files. 
I couldn’t access the git clone ssh://git@stash.devlan.net:7999/~User #524297/git-prune-files.git
It says 

fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

I will try again to “rewrite the history”. 
Thanks, 
Payam

**Comment by Besler** [comment]
Oh sorry! That was an example of how to clean up the git repo.

This looks great though! Thank you for removing the MHD data. Later tonight I'll commit my visualization code.

**Review by Besler — APPROVED** [review]

**Comment by Besler** [comment]
Since you created the PR, you merge it into master. You get that sweet, sweet satisfaction!

I would recommend deleting the remote branch after. GitHub will just have a click button thing. 'Delete branch?'

**Comment by PayamZandiyeh** [comment]
I have done as suggested. 
Thanks man! 
Cheers, 
Payam

**Pull request merged**